Bug: New Jupyterhub image fails to deploy on Kubernetes - helm upgrade

[Github Action](https://github.com/cal-itp/data-infra/actions/runs/16273687251/job/45956618232)

I ran it several times after https://github.com/cal-itp/data-infra/pull/4075.

Jupyterhub is still running fine but this will block us from deploying new images.
```
Run poetry run invoke release -f "./kubernetes-workloads.yaml"
  poetry run invoke release -f "./kubernetes-workloads.yaml"
  shell: /usr/bin/bash -e ***0***
  env:
    GITHUB_TOKEN: ***
    CLOUDSDK_CORE_PROJECT: cal-itp-data-infra
    GKE_NAME: data-infra-apps
    GKE_REGION: us-west1
    USE_GKE_GCLOUD_AUTH_PLUGIN: true
    CLOUDSDK_AUTH_CREDENTIAL_FILE_OVERRIDE: /home/runner/work/data-infra/data-infra/gha-creds-b03577f1d464b8d2.json
    GOOGLE_APPLICATION_CREDENTIALS: /home/runner/work/data-infra/data-infra/gha-creds-b03577f1d464b8d2.json
    GOOGLE_GHA_CREDS_PATH: /home/runner/work/data-infra/data-infra/gha-creds-b03577f1d464b8d2.json
    CLOUDSDK_PROJECT: cal-itp-data-infra
    GCLOUD_PROJECT: cal-itp-data-infra
    GCP_PROJECT: cal-itp-data-infra
    GOOGLE_CLOUD_PROJECT: cal-itp-data-infra
    CLOUDSDK_METRICS_ENVIRONMENT: github-actions-setup-gcloud
    CLOUDSDK_METRICS_ENVIRONMENT_VERSION: 2.1.4
    KUBECONFIG: /home/runner/work/data-infra/data-infra/gha-kubeconfig-abe5b486df3b4fd1
    KUBE_CONFIG_PATH: /home/runner/work/data-infra/data-infra/gha-kubeconfig-abe5b486df3b4fd1
Executing: kubectl apply -k /home/runner/work/data-infra/data-infra/kubernetes/apps/overlays/gtfs-rt-archiver-v3-prod
namespace/gtfs-rt-v3 unchanged
configmap/archiver-app-vars unchanged
configmap/archiver-channel-vars unchanged
service/redis unchanged
deployment.apps/gtfs-rt-archiver-consumer configured
deployment.apps/gtfs-rt-archiver-ticker configured
deployment.apps/redis configured
Executing: kubectl apply -k /home/runner/work/data-infra/data-infra/kubernetes/apps/manifests/airflow-jobs
namespace/airflow-jobs unchanged
priorityclass.scheduling.k8s.io/dbt-high-priority configured
Executing: helm dependency update /home/runner/work/data-infra/data-infra/kubernetes/apps/charts/grafana
Executing: helm upgrade grafana /home/runner/work/data-infra/data-infra/kubernetes/apps/charts/grafana --install --namespace monitoring-grafana --values=/home/runner/work/data-infra/data-infra/kubernetes/apps/values/grafana.yaml --values=/home/runner/work/data-infra/data-infra/kubernetes/apps/values/grafana-prod.yaml --reset-values 
Release "grafana" has been upgraded. Happy Helming!
NAME: grafana
LAST DEPLOYED: Mon Jul 14 19:58:03 2025
NAMESPACE: monitoring-grafana
STATUS: deployed
REVISION: 91
TEST SUITE: None
NOTES:
1. Get your 'admin' user password by running:

   kubectl get secret --namespace monitoring-grafana grafana -o jsonpath="***.data.admin-password***" | base64 --decode ; echo

2. The Grafana server can be accessed via port 80 on the following DNS name from within your cluster:

   grafana.monitoring-grafana.svc.cluster.local

   If you bind grafana to 80, please update values in values.yaml and reinstall:

   securityContext:
     runAsUser: 0
     runAsGroup: 0
     fsGroup: 0

   command:
   - "setcap"
   - "'cap_net_bind_service=+ep'"
   - "/usr/sbin/grafana-server &&"
   - "sh"
   - "/run.sh"

   Details refer to https://grafana.com/docs/installation/configuration/#http-port.
   Or grafana would always crash.

   From outside the cluster, the server URL(s) are:
     http://monitoring.k8s.calitp.jarv.us/
     http://monitoring.calitp.org/

3. Login with the password from step 1 and the username: admin
Executing: helm dependency update /home/runner/work/data-infra/data-infra/kubernetes/apps/charts/jupyterhub
Getting updates for unmanaged Helm repositories...
...Successfully got an update from the "https://jupyterhub.github.io/helm-chart" chart repository
Saving 1 charts
Downloading jupyterhub from repo https://jupyterhub.github.io/helm-chart
Deleting outdated charts
Downloaded secret helm values: /tmp/tmp50c6v1y3/jupyterhub_jupyterhub-sensitive-helm-values.yaml
Error: UPGRADE FAILED: pre-upgrade hooks failed: 1 error occurred:
	* timed out waiting for the condition


Executing: helm upgrade jupyterhub /home/runner/work/data-infra/data-infra/kubernetes/apps/charts/jupyterhub --install --namespace jupyterhub --values=/tmp/tmp50c6v1y3/jupyterhub_jupyterhub-sensitive-helm-values.yaml --reset-values --timeout 10m0s
Error: Process completed with exit code 1.

   ```

--- a/ci/kubernetes-workloads.yaml
+++ b/ci/kubernetes-workloads.yaml
@@ -31,7 +31,7 @@ calitp:
       secrets:
         - jupyterhub_jupyterhub-gcloud-service-key
         - jupyterhub_jupyterhub-github-config
-      timeout: 15m0s
+      timeout: 10m0s
     - name: loki
       driver: helm
       namespace: monitoring-loki

--- a/kubernetes/apps/charts/jupyterhub/values.yaml
+++ b/kubernetes/apps/charts/jupyterhub/values.yaml
@@ -40,17 +40,17 @@ jupyterhub:
           mem_limit: "12G"
           mem_guarantee: "10G"
           cpu_guarantee: 1.5
-      - display_name: "Prototype Image - 2024.6.10, Python 3.11"
+      - display_name: "Prototype Image - 2025.6.10, Python 3.11"
         description: "This is the newer environment for new package testing."
         kubespawner_override:
-          image: ghcr.io/cal-itp/data-infra/jupyter-singleuser:2024.6.10
-      - display_name: "Power Prototype Image - 2024.6.10, Python 3.11"
+          image: ghcr.io/cal-itp/data-infra/jupyter-singleuser:2025.6.10
+      - display_name: "Power Prototype Image - 2025.6.10, Python 3.11"
         description: "Your code will run on a shared machine with 10–12G of memory and 1.5–1.9 CPU cores."
         kubespawner_override:
           mem_limit: "12G"
           mem_guarantee: "10G"
           cpu_guarantee: 1.5
-          image: ghcr.io/cal-itp/data-infra/jupyter-singleuser:2024.6.10
+          image: ghcr.io/cal-itp/data-infra/jupyter-singleuser:2025.6.10
   scheduling:
     userPods:
       nodeAffinity:

